src/components/about/Clients.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Image from "next/image";
+import talo from "../../../public/assets/images/about/clients/talo.png";
+import solidState from "../../../public/assets/images/about/clients/solid-state.png";
+import noted from "../../../public/assets/images/about/clients/noted.png";
+import goan from "../../../public/assets/images/about/clients/goan.png";
+import mowi from "../../../public/assets/images/about/clients/mowi.png";
```

Comment:
dont import images this way, just reference its path relative to public dir

e.g. 
```jsx
<Image
  src="/assets/images/about/clients/noted.png"
  ...
/>
```
or store the strings in variables
```js
const notedImage = "/assets/images/about/clients/noted.png"
```